src/audio/dai-zephyr.c

```diff
@@ -452,7 +452,7 @@ static int dai_playback_params(struct comp_dev *dev, uint32_t period_bytes,
 		*local_buf = buffer_acquire(dd->local_buffer);
```

Comment:
@abonislawski Can you update the commit message a bit as well and maybe add tag for the bug, i.e. "Link: https://github.com/thesofproject/sof/issues/6709" ?

Reply:
Updated
PR linked to 6709